### PR TITLE
Fix link to header and add homepage link to top of table of contents

### PR DESCRIPTION
**Pull request by samjwu**
fix header link to amd.com

~~add 2nd level header link to rocm.docs.amd.com~~ add back homepage link (ROCm Documentation Home) to top of table of contents

### CODE REVIEW

**Review by saadrahim — APPROVED** [review]

File: src/rocm_docs/rocm_docs_theme/components/left-side-menu.html [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<a class="navbar-brand logo" href="https://rocm.docs.amd.com/en/latest/">
```
Comment (saadrahim) [inline src/rocm_docs/rocm_docs_theme/components/left-side-menu.html]: we need this to be dynamic based on branch

**Review by saadrahim — COMMENTED** [review]
see comment

**Review by saadrahim — COMMENTED** [review]
This might not work if the project is on amd-master or master, etc.

**Comment by samjwu** [comment]
> This might not work if the project is on amd-master or master, etc.

As a compromise, acab162 will dynamically change homepage link for develop branch and use the link to latest for other branches to avoid a 404 error

File: src/rocm_docs/rocm_docs_theme/components/left-side-menu.html (hunk 1 above)
Comment (samjwu) [reply src/rocm_docs/rocm_docs_theme/components/left-side-menu.html]: Done in ccda756

**Comment by samjwu** [comment]
Once there is a common convention (eg: docs/5.0), we can include those in a YAML file as a sort of approve list.

eg:
jinja
```jinja
{% if branch in valid %}
<a class="navbar-brand logo" href="https://rocm.docs.amd.com/en/{{ branch }}"></a>
{% endif %}
```

yaml
```yaml
branch: master
valid: 
- develop 
- master
```

File: src/rocm_docs/rocm_docs_theme/components/left-side-menu.html [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {% macro navbar_item(branch) -%}
-    {% if branch in ["develop", "amd-master", "amd-staging", "master"] %}
```
Comment (saadrahim) [inline src/rocm_docs/rocm_docs_theme/components/left-side-menu.html]: I think you want the master branch as well

Comment (samjwu) [reply src/rocm_docs/rocm_docs_theme/components/left-side-menu.html]: Added back in [2c26f64](https://github.com/RadeonOpenCompute/rocm-docs-core/pull/211/commits/2c26f644f777d403351f556b8762ef54f8f78cfe)

**Comment by saadrahim** [comment]
much nicer, but still a few things to think about.

**Comment by saadrahim** [comment]
https://www.compart.com/en/unicode/U+2302 ROCm Docs?
Would that work?

**Comment by samjwu** [comment]
> https://www.compart.com/en/unicode/U+2302 ROCm Docs? Would that work?

I've made the following changes:
- ROCm Documentation Home > &#x2302; ROCm Docs
- left align all table of contents (the ReadtheDocs build uses the files in rocm_docs_theme instead of the custom.css file in the rocm_docs_theme folder I think, which is why it does not appear left aligned in the PR preview site)

**Comment by saadrahim** [comment]
You are going to have to amend my commit. Commitzen catches off guard to often.

**Review by saadrahim — APPROVED** [review]

**Pull request merged**